### PR TITLE
Rankin/readme

### PR DESCRIPTION
**Pull request by rankind94**
Updating the README file.

### CODE REVIEW

**Review by santosb93 — APPROVED** [review]

**Review by ItsMingNow — APPROVED** [review]
Great! Approved!

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# main
```
Comment (zackdaniels) [inline README.md:1]: Approved

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+# Welcome to our e-Ventory
```
Comment (zackdaniels) [inline README.md:1]: Approved

**Review by zackdaniels — APPROVED** [review]
Approved

**Review by santosb93 — APPROVED** [review]

**Review by zackdaniels — APPROVED** [review]

**Review by zackdaniels — APPROVED** [review]

**Review by santosb93 — APPROVED** [review]

**Review by ItsMingNow — APPROVED** [review]
Good! Approved!

**Review by santosb93 — APPROVED** [review]

**Review by zackdaniels — APPROVED** [review]

**Review by ItsMingNow — APPROVED** [review]
Great! Approved!!

**Review by santosb93 — APPROVED** [review]

**Review by zackdaniels — APPROVED** [review]

**Pull request merged**